apps/src/lib/script-editor/LessonExtrasEditor.jsx

```diff
@@ -35,20 +36,21 @@ export default class LessonExtrasEditor extends React.Component {
   render() {
     return (
       <div>
-        <h3> Lesson Extras Settings </h3>
         <label>
-          Lesson Extras Available
+          Allow Teachers to Enable Lesson Extras
           <input
             name="lesson_extras_available"
             type="checkbox"
             checked={this.state.lessonExtras}
             style={styles.checkbox}
             onChange={e => this.setState({lessonExtras: e.target.checked})}
           />
-          <p>
-            If also enabled by the teacher, show the lesson extras page at the
-            end of each lesson.
-          </p>
+          <HelpTip>
+            <p>
```

Comment:
just curious if it looks any different/worse without the `<p>`, like this: https://github.com/code-dot-org/code-dot-org/blob/35718de683cbf67fa2761c63bf0f2db3b54fcd44/apps/src/lib/ui/HelpTip.story.jsx#L10

Reply:
I checked and it seems to work fine without the paragraph tag as long as all the text is supposed to be smushed together (not in 2 paragraphs or something). I'm going to leave it for now.